src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -45,134 +45,185 @@
 // * Copyright 2010 by Ember Corporation. All rights reserved.              *80*
 // *******************************************************************
 #include "groups-server.h"
-
 #include <app/CommandHandler.h>
+#include <app/clusters/scenes/scenes.h>
 #include <app/common/gen/att-storage.h>
 #include <app/common/gen/attribute-id.h>
 #include <app/common/gen/attribute-type.h>
+#include <app/common/gen/attributes/Accessors.h>
 #include <app/common/gen/cluster-id.h>
 #include <app/common/gen/command-id.h>
 #include <app/util/af.h>
-#include <app/util/binding-table.h>
-
-#ifdef EMBER_AF_PLUGIN_SCENES
-#include <app/clusters/scenes/scenes.h>
-#endif // EMBER_AF_PLUGIN_SCENES
+#include <map>
```

Comment:
I think this may compile/run in embedded. I believe we do not allow std::map/std::set on those.